deployer/src/deployment/run.rs

```diff
@@ -348,10 +354,11 @@ async fn load(
     match response {
         Ok(response) => {
             let response = response.into_inner();
+            // Make sure to not log the entire response, the resources field is likely to contain secrets.
+            if response.success {
+                info!("successfully loaded service");
+            }
```

Comment:
It seems this comment can now safely be removed

Reply:
It's more of a reminder to not print the entire response, since it does still contain the resources.